src/pybind/mgr/cephadm/module.py

```diff
@@ -1953,6 +1953,7 @@ def _apply_service_spec(self, spec: ServiceSpec) -> str:
                 'mgr': PlacementSpec(count=2),
                 'mds': PlacementSpec(count=2),
                 'rgw': PlacementSpec(count=2),
+                'haproxy': PlacementSpec(count=2),
```

Comment:
I think it should be :  
'ha_rgw' instead 'haproxy'
because:
'haproxy' is just the name of the load balancer we are using, and the thing we want to have is "high availability for rgw service". In the future we can implement other services ... for example ha_dashboard, ha_grafana, ha_prometheus, etc ...
In the pr for documentation i have used "ha_rgw" for this reason